BookingSystem/BookingSystem.Queries/Queries/CountryQueries/Queries/CountryDetailsQuery.cs

```diff
@@ -1,14 +1,11 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+﻿using System.Linq;
 using System.Threading.Tasks;
 using BookingSystem.Common.Interfaces;
-using BookingSystem.Queries.Views;
+using BookingSystem.Queries.Queries.CountryQueries.Views;
 using BookingSystem.ReadPersistence;
 using Microsoft.EntityFrameworkCore;
 
-namespace BookingSystem.Queries.CountryQueries
+namespace BookingSystem.Queries.Queries.CountryQueries.Queries
 {
     public class CountryDetailsQuery : IQuery<Task<CountryView>>
```

Comment:
Can you please remind me what case will be when you need the single city/country details?